### PR TITLE
feat: formatting, chapter restructering, etc.

### PR DESCRIPTION
**Pull request by stockbal**
- Adds child pages to chapter 7 (Plug-ins)
- Adds link color during hover
- Adds image chapter formatting
- Adds image shadow to improve visibility
- Add some translations to theme defaults like search field and table of contents

### CODE REVIEW

File: docs/plug-ins/open-source-adt-plugins.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
+---
+layout: page
+title: Nützliche Open Source ADT Plugins
+parent: Kapitel 7 - Plug-ins
+permalink: /plug-ins/open-source-adt-plugins/
+nav_order: 2
+---
+
+1. TOC
+{:toc}
```
Comment (marianfoo) [inline docs/plug-ins/open-source-adt-plugins.md:10]: I would use the toc after the main heading `# Nützliche Open Source ADT Plugins`
You can exclude the main heading with `{: .no_toc }`
https://just-the-docs.github.io/just-the-docs/docs/navigation-structure/#in-page-navigation-with-table-of-contents

**Review by marianfoo — CHANGES_REQUESTED** [review]
small suggestions

File: docs/_sass/custom/custom.scss [hunk 2]
```diff
@@ -1,21 +1,40 @@
...
 
 .underline {
-    text-decoration: underline;
-  }
\ No newline at end of file
+  text-decoration: underline;
+}
+
+a:hover {
+  text-decoration: underline #eb5c3f !important;
```
[the remark this answers is not in the document]
Comment (stockbal) [reply docs/_sass/custom/custom.scss:24]: It is actually the orange color used on the DSAG hompage 
image

Together with the blue color I used for the text of the links I think it is a good contrast

[the remark this answers is not in the document]
Comment (marianfoo) [reply docs/_sass/custom/custom.scss:24]: Ah, in this context the color choice make sense, you´re right

File: docs/plug-ins/open-source-adt-plugins.md (hunk 1 above)
Comment (stockbal) [reply docs/plug-ins/open-source-adt-plugins.md:14]: Good Idea. I updated the pull request with the requested changes

Comment (marianfoo) [reply docs/plug-ins/open-source-adt-plugins.md:14]: Looks good, would be perfect if the other chapters use this as a template and then are built similar

**Pull request merged**